src/msgs/data_requests/sudo/remove_requests.rs

```diff
@@ -1,14 +1,14 @@
 use super::*;
 
-#[cfg_attr(feature = "cosmwasm", cw_serde)]
+#[cfg_attr(feature = "cosmwasm", cosmwasm_schema::cw_serde)]
 #[cfg_attr(not(feature = "cosmwasm"), derive(serde::Serialize, Debug, PartialEq))]
 #[cfg_attr(not(feature = "cosmwasm"), serde(rename_all = "snake_case"))]
 pub struct Sudo {
-    pub results: Vec<PostResult>,
+    pub requests: Vec<RemoveDataRequest>,
```

Comment:
What is the benefit of using a vector of  `RemoveDataRequest` instead of a vector of strings?

Reply:
Mostly laziness, so I could map over and call the `RemoveDataRequest` method on each one.

Otherwise, when I iterate over the list, it would first be me mapping over to convert each string to that, then calling its method.